docs/domains/Domain.json

```diff
@@ -378,6 +378,10 @@
             "ADMIN_ONLY"
           ]
         },
+        "livenessProbeCustomScript": {
+          "description": "Full path of the liveness probe custom script",
```

Comment:
Please make it clear that this is optional, for advanced users only, and that the default is 'not set'.  Also, what happens if this script is not found?